resources/views/dashboard/controllers/roster.blade.php

```diff
@@ -1,319 +1,170 @@
 @extends('layouts.dashboard')
 
 @section('title')
-    Roster
+Roster
 @endsection
 
-@section('content')
-    <div class="container-fluid" style="background-color:#F0F0F0;">
-        &nbsp;
-        <h2>Roster</h2>
-        &nbsp;
-    </div>
-    <br>
+@push('custom_header')
+<link rel="stylesheet" href="{{ asset('css/roster.css') }}" />
+@endpush
 
-    <div class="container">
-        <h5>Certification Key:</h5>
-        <div class="row">
-            <div class="col-sm-2">
-                <p>No Certification:</p>
-            </div>
-            <div class="col-sm-2">
-                <i class="fas fa-times" style="color:red"></i>
-            </div>
-        </div>
-        <div class="row">
-            <div class="col-sm-2">
-                <p>Minor Certification:</p>
-            </div>
-            <div class="col-sm-2">
-                <i class="far fa-check-circle" style="color:green"></i>
-            </div>
-        </div>
-        <div class="row">
-            <div class="col-sm-2">
-                <p>Full (Minor/Major) Certification:</p>
-            </div>
-            <div class="col-sm-2">
-                <i class="fas fa-check" style="color:green"></i>
-            </div>
-        </div>
-        <br>
-        @if(Auth::user()->isAbleTo('roster'))
-            <a href="/dashboard/admin/roster/visit/requests" class="btn btn-warning">Visit Requests</a>
-            <a href="/dashboard/admin/roster/purge-assistant" class="btn btn-danger">Roster Purge Assistant</a>
-            <span data-toggle="modal" data-target="#allowVisitor">
-            <button type="button" class="btn btn-warning">Allow Rejected Visitor</button>
-        </span>
-            <br><br>
-        @endif
-        <ul class="nav nav-tabs nav-justified" role="tablist">
-            <li class="nav-item">
-                <a class="nav-link active" href="#home" role="tab" data-toggle="tab" style="color:black">Home Controllers</a>
-            </li>
-            <li class="nav-item">
-                <a class="nav-link" href="#visit" role="tab" data-toggle="tab" style="color:black">Visiting Controllers</a>
-            </li>
-        </ul>
-        <div class="tab-content">
-            <div role="tabpanel" class="tab-pane active" id="home">
+@section('content')
```

Comment:
This is so, so much better than a grid of checkboxes. Good work!

Reply:
Thanks!